lib/authsense/service.ex

```diff
@@ -85,10 +93,12 @@ defmodule Authsense.Service do
 
       get_user("rico@gmail.com")  #=> %User{...}
   """
-  def get_user(email, model \\ nil) do
+  def get_user(email, model \\ nil, opts \\ []) do
```

Comment:
Hmm, I'd be wary of having 2 optional args (`get_user(email, model\\nil, opts\\[])`) because that means you can't specify `opts` without specifying `model`.

How about transitioning to a new signature `get_user(email, opts)`, while retaining support for the old model:

```
# Support old signature
def get_user(email, model) when is_atom(model), do:
  get_user(email, model: model)

# New signature
def get_user(email, opts \\ []), do:
  get_user(email, opts)
```

This way you can do:

```ex
# Old sig
get_user("hi@gmail.com", User)

# New sig
get_user("hi@gmail.com", foo: :bar)
get_user("hi@gmail.com", model: User, foo: :bar)
```